src/app.ts

```
@@ -0,0 +1,24 @@
+import express from 'express';
+import cors, { CorsOptions } from 'cors';
+import { configDotenv } from 'dotenv';
+import router from './controller';
+
+configDotenv();
+const port: Number = Number(process.env.PORT!) ?? 8080;
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)